src/docs/sphinx/BuildingAscent.rst

```diff
@@ -13,9 +13,8 @@ This page provides details on several ways to build Ascent from source.
 
 For the shortest path from zero to Ascent, see :doc:`QuickStart`.
 
-To build third party dependencies we recommend using :ref:`uberenv <building_with_uberenv>` which leverages Spack or :ref:`Spack directly<building_with_spack>`.
-We also provide info about :ref:`building for known HPC clusters using uberenv <building_known_hpc>`.
-and a :ref:`Docker example <building_with_docker>` that leverages Spack.
+To build third party dependencies we recommend using :ref:`build_ascent.sh`, :ref:`uberenv <building_with_uberenv>` which leverages Spack, or :ref:`Spack directly<building_with_spack>`. We also provide a :ref:`Docker example <building_with_docker>` that leverages Spack.
+.. info about :ref:`building for known HPC clusters using uberenv <building_known_hpc>`. and
```

Comment:
I think this is an unfinished thought? broken reference and I think also wants to point at build_ascent.sh

Reply:
last part os old context I moved and commented out, I'll re-review as rendered.

Reply:
Ah gotcha, weird that it rendered for me.

Reply:
looks like i botched the spacing for the comment,thanks!